packages/indexer/src/services/deposits.ts

```
@@ -293,11 +293,12 @@ export async function Indexer(config: Config) {
       }),
     );
 
+  const dbThrowError = false;
```

What's the reason for silencing the errors? Don't we want the events to be scraped again if for exemple there was a problem in the storage layer?

---

This is for avoiding errors while developing as we don't have the indexing loop in place yet. It should be temporary, I'll add a TODO comment to update this when we improve the indexing loop.